app/assets/javascripts/discourse/tests/unit/lib/uploads-test.js

```diff
@@ -138,6 +138,38 @@ discourseModule("Unit | Utility | uploads", function () {
     );
   });
 
+  test("shows error message when no extensions are authorized", function (assert) {
+    this.siteSettings.authorized_extensions = "";
+    this.siteSettings.authorized_extensions_for_staff = "";
```

Comment:
```suggestion
    this.siteSettings.authorized_extensions = "";
```

I think we only need to change this site setting for this test case.